lib/plugin/instance.rb

```diff
@@ -74,6 +74,11 @@ def seed_fu_filter(filter = nil)
     @seed_fu_filter = filter
   end
 
+  # This method returns Core stats + stats registered by plugins
+  def self.get_stats
```

Comment:
Do we need get_ ? doesnt feel very ruby to me

Reply:
Yeah, I was on the fence myself with this, kind of thought that for an API method it _may_ make sense? But let's better go without `get`, if it feels weird for you too.